res/lang/news/news_es.txt

```diff
@@ -77,6 +77,7 @@ NEWS_ANNOUNCE_MOVIE_DESCRIPTION1 = Director %DIRECTORNAME1% and actor %ACTORNAME
 NEWS_ANNOUNCE_MOVIE_DESCRIPTION2 = Soon in the movies: %ACTORNAME1% in «%MOVIETITLE%». Actor %ACTORNAME2% is full of praise for the movie.
 NEWS_ANNOUNCE_MOVIE_DESCRIPTION3 = Latest works for movie «%MOVIETITLE%». %ACTORNAME1% is proud starring the movie.
 NEWS_ANNOUNCE_MOVIE_DESCRIPTION4 = First trailers for «%MOVIETITLE%» available. %DIRECTORNAME1%: "Best movie of the year".
+#NEWS_ANNOUNCE_MOVIE_DESCRIPTION5 =
```

Comment:
Did you add a new announcement to de an en as well (not part of the last commit)? Otherwise these entries have to be removed for the property files to be in sync.

Reply:
Sorry, I checked only the latest commit. Missed the additions in the previous one.